kmip/services/server/engine.py

```diff
@@ -1556,3 +1561,61 @@ def _process_discover_versions(self, payload):
         )
 
         return response_payload
+
+    @_kmip_version_supported('1.2')
+    def _process_mac(self, payload):
+        self._logger.info("Processing operation: MAC")
+
+        unique_identifier = self._id_placeholder
+        if payload.unique_identifier:
+            unique_identifier = payload.unique_identifier.value
+
+        object_type = self._get_object_type(unique_identifier)
+
+        managed_object = self._data_session.query(object_type).filter(
+            object_type.unique_identifier == unique_identifier
+        ).one()
```

Comment:
So that's the new stuff~ But I met an issue.  It seems that the operation policy(3.18) doesn't control operations like MAC which should be restricted by crypto masks(as we discussed before). That's why in my codes I didn't check policy in mac operation. In the new function, _is_allowed_by_operation_policy is called anyway. So do you think I should add MAC into policy.py or keep my current codes?